inventory

```diff
@@ -1,20 +1,62 @@
 [OSEv3:children]
 masters
 nodes
+glusterfs
 
 [OSEv3:vars]
 openshift_deployment_type=origin
 containerized=true
-openshift_image_tag=v3.7.0
+openshift_image_tag=v3.9.0-alpha.4
```

Comment:
@aglitke Are you sure about 3.9 here? ATM, AFAIK, kubevirt-ansible defaults to 3.7. Please see issue #147